src/Controller/Admin/Document/DocumentController.php

```diff
@@ -755,13 +755,13 @@ public function publishVersionAction(Request $request): JsonResponse
      */
     public function updateSiteAction(Request $request): JsonResponse
     {
-        $domains = $request->get('domains');
+        $domains = $request->request->get('domains', '');
         $domains = str_replace(' ', '', $domains);
-        $domains = explode("\n", $domains);
+        $domains = $domains ? explode("\n", $domains) : [];
```

Comment:
what about 
```
        $domains = $request->request->get('domains');
        if ($domains){
           if (is_string($domains)){
               $domains = str_replace(' ', '', $domains);
               $domains = $domains ? explode("\n", $domains) : [];
           }
        }else{
           $domains = [];
        }
```
?

Reply:
@kingjia90 I changed it to getString(), so it must be a string. See https://github.com/pimcore/admin-ui-classic-bundle/pull/581/commits/f616ff80df6407638922ca17e0c8b89d7ab36e83